packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveMetadata/forceSourceRetrieveCmp.ts

```diff
@@ -21,7 +21,7 @@ import {
   SourceComponent
 } from '@salesforce/source-deploy-retrieve';
 import { SourceRetrieveResult } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
-import { ComponentLike } from '@salesforce/source-deploy-retrieve/lib/src/common/types';
+import { ComponentLike } from '@salesforce/source-deploy-retrieve/lib/src/resolve/types';
```

Comment:
Not for this PR, but we should expose these types (`SourceRetrieveResult`, `ComponentLike`, etc) at the top level of the library so we can import them just using `@salesforce/source-deploy-retrieve`.